lib/pages/login/login_model.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_survey/api/exception/network_exceptions.dart';
+import 'package:flutter_survey/pages/login/login_state.dart';
+import 'package:flutter_survey/usecase/base/base_use_case.dart';
+import 'package:flutter_survey/usecase/login_use_case.dart';
+
+class LoginModel extends StateNotifier<LoginState> {
```

Comment:
Model sounds like a normal `Model` class in any architecture: M(odel)VVM, M(odel)VP...🤔  shouldn't you consider `LoginViewModel` like we always do? 🤷‍♂️

Reply:
Fixed https://github.com/luongvo/flutter-survey/pull/51/commits/5d4d5216ce5dc0496ab1af80707465e19f95e0c6